analysis/report_templates/default.html

```diff
@@ -43,10 +43,10 @@ <h5 class="center-align brown-text text-darken-4">
 
         <div class="card-panel amber lighten-5">
             <div class="row">
-                <div class="col s1">
-                    <i class="medium material-icons">warning</i>
+                <div class="col s4 m2">
+                    <center><i class="medium material-icons">warning</i></center>
```

Comment:
does "center-align" inside class work, i think that is how is done in many places.

Reply:
Yes! That's much better indeed, thanks!